dev/node_app/src/app.ts

```diff
@@ -86,20 +86,20 @@ const commandBindings = {
         {
             icon: 'icon.png',
             label: 'node-example',
-            description: 'Example App written with Node.js',
+            description: 'Example app written with Node.js',
             hint: '[send]',
             bindings: [
                 {
                     location: 'send',
                     label: 'send',
-                    form,
+                    form
```

Comment:
It's better convention to have the dangling comma here

```suggestion
                    form,
```